### PR TITLE
FSE: Fix lint errors flagged by PHPCS

### PR DESCRIPTION
**Pull request by ockham**
#### Changes proposed in this Pull Request

When trying to deploy the FSE plugin to WP.com, SVN's pre-commit hooks complained about a number of lints. This PR fixes them.

As a follow-up, we should make sure that these get caught earlier -- ideally all of the following stages should run checks:
- Locally: on a pre-commit git hook
- Calypso: in CI
- TeamCity

#### Testing instructions

Run PHPCS on the `patterns` subdir.

### CODE REVIEW

File: apps/full-site-editing/full-site-editing-plugin/block-patterns/patterns/contact-04.php [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
 <!-- wp:group {"align":"full","style":{"color":{"background":"#000000"}}} -->
 <div class="wp-block-group alignfull has-background" style="background-color:#000000"><div class="wp-block-group__inner-container"><!-- wp:social-links {"align":"center"} -->
-<ul class="wp-block-social-links aligncenter"><!-- wp:social-link {"url":"","service":"wordpress"} /-->
+<ul class="wp-block-social-links aligncenter"><!-- wp:social-link {"url":"","service":"WordPress"} /-->
```
Comment (ockham) [inline apps/full-site-editing/full-site-editing-plugin/block-patterns/patterns/contact-04.php:43]: Uh crap, PHPCBF did this. I'll fix it.

**Comment by ockham** [comment]
FWIW, pretty much all of these are instances of

```
 76 | ERROR   | [ ] Trailing comma's are not allowed in function calls in PHP 7.2 or earlier
    |         |     (PHPCompatibility.Syntax.NewFunctionCallTrailingComma.FoundInFunctionCall)
 77 | ERROR   | [ ] PHP syntax error: syntax error, unexpected ')' (Generic.PHP.Syntax.PHPSyntax)
```

File: apps/full-site-editing/full-site-editing-plugin/block-patterns/patterns/contact-04.php (hunk 1 above)
Comment (ockham) [reply apps/full-site-editing/full-site-editing-plugin/block-patterns/patterns/contact-04.php]: 1e7044a

Comment (noahtallen) [reply apps/full-site-editing/full-site-editing-plugin/block-patterns/patterns/contact-04.php]: lovely! thanks

**Comment by ockham** [comment]
Thanks for reviewing, y'all!

**Comment by vindl** [comment]
> Locally: on a pre-commit git hook

This was already introduced. I guess people were committing with `--no-verify`.

**Pull request merged**